constance/backends/database/__init__.py

```diff
@@ -87,7 +86,7 @@ def set(self, key, value):
         if self._cache:
             self._cache.set(key, value)
 
-        updated_signal.send(sender='constance', key=key, value=value)
+        signals.config_updated.send(sender='constance', key=key, value=value)
```

Comment:
How should I get the `config` object in this file?

I'm guessing I should do something like:
```
from constance import config
...
signals.config_updated.send(sender=config)
```

In case you want to see the other parameters should I rename them to `updated_key` and `new_value` ?
```
signals.config_updated.send(sender=config, updated_key=key, new_value=value)
```